crates/miden-lib/src/errors/tx_kernel_errors.rs

```diff
@@ -160,7 +160,7 @@ pub const ERR_NOTE_INVALID_INDEX: MasmError = MasmError::from_static_str("failed
 /// Error Message: "invalid note type for the given note tag prefix"
 pub const ERR_NOTE_INVALID_NOTE_TYPE_FOR_NOTE_TAG_PREFIX: MasmError = MasmError::from_static_str("invalid note type for the given note tag prefix");
 /// Error Message: "the specified number of note inputs does not match the actual number"
-pub const ERR_NOTE_INVALID_NUMBER_OF_NOTE_INPUTS: MasmError = MasmError::from_static_str("the specified number of note inputs does not match the actual number");
+pub const ERR_NOTE_INVALID_NUMBER_OF_NOTE_PAYLOAD: MasmError = MasmError::from_static_str("the specified number of note inputs does not match the actual number");
```

Comment:
Similarly as with other comments, we should refer to this as payload length instead of "number of payload".